platform/src/main/java/org/stellar/anchor/platform/configurator/DataConfigAdapter.java

```diff
@@ -131,6 +131,9 @@ void updateSpringEnv(ConfigMap config) throws InvalidConfigException {
           set("spring.flyway.user", SecretManager.getInstance().get(SECRET_DATA_USERNAME));
           set("spring.flyway.password", SecretManager.getInstance().get(SECRET_DATA_PASSWORD));
           set("spring.flyway.url", constructPostgressUrl(config));
+          set(
+              "spring.flyway.baseline-on-migrate",
+              config.getString("data.flyway_baseline_on_migrate").equalsIgnoreCase("true"));
```

Comment:
🚨  I've tested it locally and got an error where the [V1_master](https://github.com/stellar/java-stellar-anchor-sdk/blob/6b57d67d9138b9cde78bbd1d8750e356e53a3dc6/platform/src/main/resources/db/migration/V1__master.sql) migration was ignored.

To fix that, you additionally need to tell Flyway what version should be given to the baseline. The official documentation suggests setting it to 0: https://flywaydb.org/documentation/configuration/parameters/baselineVersion

According to [the documentation](https://flywaydb.org/documentation/configuration/parameters/baselineOnMigrate), _"Only migrations above `baselineVersion` will then be applied."_.

Reply:
Here's the fix that worked for me:
```suggestion
          boolean baselineOnMigrate =
              config.getString("data.flyway_baseline_on_migrate").equalsIgnoreCase("true");
          if (baselineOnMigrate) {
            set("spring.flyway.baseline-on-migrate", true);
            set("spring.flyway.baseline-version", "0");
          }
```

It should be fine to default the baseline-version to `0` when baseline-on-migrate is present.